### ISSUE TITLE
Test for WCAG success criterion 3.2.1:  On Focus

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#214]
http://www.w3.org/WAI/WCAG20/quickref/#qr-consistent-behavior-receive-focus

When any component receives focus, it does not initiate a change of context.
## Applicablity

Always
## Failures
- [ ] F52: Opening a new window as soon as a new page is loaded
- [ ] F55: Using script to remove focus when focus is received 
## Technique
- [x] G107 (#148): Using "activate" rather than "focus" as a trigger for changes of context
